reg/correctMotion_DFT.m

```diff
@@ -0,0 +1,64 @@
+function correctMotion_DFT(hdf5_dir,hdf5_in_name,hdf5_out_name)
+
+dataset = '/Data/Images';
+
+h5att = h5info(fullfile(hdf5_dir,hdf5_in_name),dataset);
```

Comment:
What's the reason for separating `hdf5_dir` and `hdf5_in_name`? For instance, in `hdf5_in_name`, you can use relative or absolute paths (e.g. `path/to/movie.hdf5`) that allows one to specify files in other directories.